javascript-source/commands/customCommands.js

```diff
@@ -203,6 +203,21 @@
             message = $.replace(message, '(count)', $.inidb.get('commandCount', event.getCommand()));
         }
 
+        if (message.match(/\(keywordcount\s(.+)\)/g)) {
+            var input_keyword = message.match(/\(keywordcount\s(.+)\)/)[1],
+                keyword_info = JSON.parse($.inidb.get('keywords', input_keyword));
+        
+            if('count' in keyword_info){
```

Comment:
Please keep the same code style as above.  Space after if (), space before braces.

Reply:
Thanks so much for your feedback. I have updated accordingly.